src/boxGrid/boxGridBox.tsx

```diff
@@ -17,16 +19,41 @@ const BoxGridBox = ({
   color = 'gray',
   image,
   actions,
+  url,
 }: Props): JSX.Element => {
-  return (
-    <div className={classNames('BoxGridBox', `BoxGridBox--${color}`)}>
+  const children = (
+    <>
       {topLabel && <div className="BoxGridBox__top-label">{topLabel}</div>}
       {image && <div className="BoxGridBox__image">{image}</div>}
       <div className="BoxGridBox__label">{label}</div>
       <div className="BoxGridBox__bottom-text">{bottomText}</div>
       {actions && <div className="BoxGridBox__actions">{actions}</div>}
-    </div>
+    </>
   );
+
+  const rootProps = {
+    className: classNames(
+      'BoxGridBox',
+      `BoxGridBox--${color}`,
+      !!url && 'BoxGridBox--link'
+    ),
+    children: children,
+  };
+
+  if (url) {
+    return createElement(
```

Comment:
Using createElement is a bit more manual than I envisioned :D The way I've usually done it is just like

```
const Component = url ? Link : Fragment;
const props = url ? { /* ... */ } : {};

<Component {...props}>
  {/* ... */}
</Component>
```
(The variable has to be capitalised to be recognised as a component rather than a HTML element.)

Sure, it works this way too, it's just untypical in most cases :)